pkg/deployment/agent.go

```diff
@@ -140,6 +140,17 @@ func (a *Agent) Get() *appsv1.DaemonSet {
 								Name:          "admin-http",
 							},
 						},
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path: "/",
+									Port: intstr.FromInt(int(adminPort)),
+								},
+							},
+							InitialDelaySeconds: 1,
+							PeriodSeconds:       15,
```

Comment:
Any particular reason for selecting 15 rather than using the default 10?

Reply:
idea was to wait little bit more and give sometime for the application to try to recover or fail on it's own before the liveliness probe intervenes and kills the pod. Decision was mostly influenced by similar experiences to what is discussed [here](https://srcco.de/posts/kubernetes-liveness-probes-are-dangerous.html)
15 * 5 = 75 s

Reply:
Ok thanks for the explanation.